index.js

```diff
@@ -0,0 +1,57 @@
+/* eslint-disable */
+import { DateTime } from "./modules/luxon.js";
+import  Book  from "./modules/books.js";
+import Display  from "./modules/bookDesplay.js";
+
+let display;
+const today = document.getElementById('today');
+
+window.onload = () => {
+  // eslint-disable-next-line no-undef
+  display = new Display();
+  display.render();
+  today.textContent = new Date().toDateString();
+};
+
```

Comment:
- [ ] According to the requirements, you should display the time in your application by using the `DateTime` object from the `Luxon` library. Kindly implement this feature. 👍